qemu/tests/numa_maxnodes.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+from qemu.tests import numa_memdev_options
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    [Memory][Numa]binding 128 guest nodes to node0 - x86 witn 32M and Power
+    with 256M, this case will:
+    1) Boot guest with 128 numa nodes
+    2) Check query-memdev
+    3) Check memory in procfs on host
+    4) Check numa node amount in guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    error_context.context("Modify params to boot guest with 128 numa nodes",
+                          logging.info)
+    node_num = int(params.get("numa_nodes", 1))
+    node_size = params["node_size"]
+    prealloc_mem = params.get("prealloc_mem", "no")
+    mem_devs = ""
+    guest_numa_nodes = ""
+    for index in range(node_num):
+        guest_numa_nodes += "node%s " % index
+        mem_devs += "mem%s " % index
+        params["numa_memdev_node%s" % index] = "mem-mem%s" % index
+        params["size_mem%s" % index] = node_size
+        params["prealloc_mem%s" % index] = prealloc_mem
+
+    params["guest_numa_nodes"] = guest_numa_nodes
+    params["mem_devs"] = mem_devs
+    params["start_vm"] = "yes"
+    env_process.preprocess_vm(test, params, env, params["main_vm"])
+    error_context.context("Get the main VM!", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
```

Comment:
This is unnecessary since you perform `wait_for_login()` immediately.

Reply:
Thanks, updated.